wbooks/src/app/screens/BookCard/styles.ts

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+import { white } from '@constants/colors';
+
+export default StyleSheet.create({
+  wrapper: {
+    backgroundColor: white,
+    borderRadius: 5,
+    flexDirection: 'row',
+    marginTop: 17,
+    height: 90,
+    width: 335
+  },
+  image: {
+    height: 60,
+    width: 40,
+    marginLeft: 30,
+    marginVertical: 15
```

Comment:
Move this (`marginVertical: 15`) to wrapper container (remember to use `padding` for internal spaces)